src/amcrest/event.py

```diff
@@ -193,21 +193,21 @@ def event_stream(self, eventcodes):
         except TypeError:
             timeout_cmd = (self._timeout_default, None)
         ret = self.command(
-            'eventManager.cgi?action=attach&codes=[{0}]'.format(eventcodes),
+            "eventManager.cgi?action=attach&codes=[{0}]".format(eventcodes),
             timeout_cmd=timeout_cmd,
-            stream=True
+            stream=True,
```

Comment:
just wondering if the , is intended here.

Reply:
The short answer is yes. The long answer is I originally put it there, since that's pretty much standard practice in Python as I understand it, but then took it out because I wasn't sure if Python2 accepted it. But then black wanted it, so I put it back. 😃